### PR TITLE
Fix leantween canceling wrong ids, fix #422, fix first person arm culling on zoom and certain viewing angles

### LINKED ISSUE #422: Zoom not always reset properly
Some deaths can cause zoom fov to be permanent

### PR DESCRIPTION
**Pull request by Fueredoriku**
After digging around for a fix for #421, I found this piece of info:
image
Which seems to be the cause of many tweening headaches (like ammoHUD not always updating correctly on high firerates), which should be resolved in this PR. I also did some small bug squashing for polish

- #421 
- #422 
- Fix first person gun arms sometimes culling on zoom and certain viewing angles

### CODE REVIEW

File: Assets/Scripts/Augment/AugmentImplementations/DDRBody.cs [hunk 1]
```diff
@@ -50,7 +50,10 @@ public class DDRBody : GunBody
     [SerializeField]
     private Precision[] precisionsGoodToBad;
 
-    LTDescr arrowMover;
+    int arrowMover;
+    int screenFlasher;
+    int screenPulseAnimator;
+    int textAnimator;
```
Comment (toberge) [inline Assets/Scripts/Augment/AugmentImplementations/DDRBody.cs:56]: These names are inconsistent with the `Tween` suffix that you've used elsewhere, perhaps rename these to match that?

File: Assets/Scripts/Control&Input/PlayerMovement.cs [hunk 2]
```diff
@@ -142,7 +147,7 @@ public void SetPlayerInput(InputManager player)
         localGunHolderX = gunHolder.transform.localPosition.x;
         localGunHolderHeight = gunHolder.transform.localPosition.y;
         playerCamera = inputManager.GetComponent<Camera>();
-        startingFov = playerCamera.fieldOfView;
+        startingFov = StaticInfo.Singleton.CameraFov;
```
Comment (toberge) [inline Assets/Scripts/Control&Input/PlayerMovement.cs:150]: This should be in a different place than the static _info_, but that can be saved for later

File: Assets/Scripts/Gamestate/PlayerManager.cs [hunk 3]
```diff
@@ -359,7 +359,7 @@ private void SetLayerOnSubtree(GameObject node, int layer)
 
     private void PlayOnHit()
     {
-        if (Random.Range(0, 1000) > 5)
+        if (Random.Range(0, 10000) > 5)
```
Comment (toberge) [inline Assets/Scripts/Gamestate/PlayerManager.cs:362]: goodbye colonel o7

**Review by toberge — APPROVED** [review]
LGTM :+1:

File: Assets/Scripts/Control&Input/PlayerMovement.cs (hunk 2 above)
Comment (Fueredoriku) [reply Assets/Scripts/Control&Input/PlayerMovement.cs:150]: Yep, I think a static settings component could be a good home for that, we'll hook it up to #383 👍

File: Assets/Scripts/Gamestate/PlayerManager.cs (hunk 3 above)
Comment (Fueredoriku) [reply Assets/Scripts/Gamestate/PlayerManager.cs:362]: 😔✊

File: Assets/Scripts/Augment/AugmentImplementations/DDRBody.cs (hunk 1 above)
Comment (Fueredoriku) [reply Assets/Scripts/Augment/AugmentImplementations/DDRBody.cs]: Fixed 👍

**Pull request merged**